src/Slider/helpers.tsx

```diff
@@ -72,3 +75,23 @@ export const startAutoplay = (
     nextImg();
   }, autoplaySpeed);
 };
+
+export const setKeyToChildren = (children: JSX.Element[]): JSX.Element[] => {
+  return children.map((child, index) => ({ ...child, key: index }));
+};
+
+export const isSlidesAnimation = (visibleCountSlides: number): boolean => {
+  return visibleCountSlides === 1;
+};
```

Comment:
Remove this function and rewrite the logic so a user doesn't have to pass additional property to set only one slide. This should be done from our side.